src/ChannelServer/Network/Sending/Send.Parties.cs

```diff
@@ -0,0 +1,447 @@
+        /// <remarks>Currently only successful</remarks>
+        /// <param name="creature"></param>
+        public static void PartyChangeFinishR(Creature creature)
+        {
+            var packet = new Packet(Op.PartyChangeFinishR, creature.EntityId);
+
+            // Can it fail?
+            packet.PutByte(true);
+            creature.Client.Send(packet);
+        }
+
+        /// <summary>
+        /// Response to leader changing EXP distribution rule.
+        /// </summary>
+        /// <remarks>Currently only successful</remarks>
+        /// <param name="creature"></param>
+        public static void PartyChangeExpR(Creature creature)
+        {
+            var packet = new Packet(Op.PartyChangeExpR, creature.EntityId);
+
+            // Can it fail?
+            packet.PutByte(true);
+            creature.Client.Send(packet);
+        }
+
+        /// <summary>
+        /// Response to leader closing the party.
+        /// </summary>
+        /// <remarks>Currently only successful</remarks>
+        /// <param name="creature"></param>
+        public static void PartyWantedClosedR(Creature creature)
+        {
+            var packet = new Packet(Op.PartyWantedHideR, creature.EntityId);
+
+            // Can it fail?
+            packet.PutByte(true);
+            creature.Client.Send(packet);
+        }
+
+        /// <summary>
+        /// Response to leader opening party.
+        /// </summary>
+        /// <remarks>Currently only successful</remarks>
+        /// <param name="creature"></param>
+        public static void PartyWantedOpenR(Creature creature)
+        {
+            var packet = new Packet(Op.PartyWantedShowR, creature.EntityId);
+
+            // Can it fail?
+            packet.PutByte(true);
+            creature.Client.Send(packet);
+        }
+
+        /// <summary>
+        /// Response to a member attempting to leave the party.
+        /// </summary>
+        /// <param name="creature"></param>
+        /// <param name="leaving"></param>
+        public static void PartyLeaveR(Creature creature, bool leaving)
+        {
+            var packet = new Packet(Op.PartyLeaveR, creature.EntityId);
+
+            packet.PutByte(leaving);
+            creature.Client.Send(packet);
+        }
+
+        // TODO: Consolidate this with AddMember functionality.
+        /// <summary>
+        /// Sent to clients when a new creature joins.
+        /// </summary>
+        /// <remarks>I'm not entirely sure of the purpose of this packet, I don't think the client would mind if you didn't send it.</remarks>
+        /// <param name="creature"></param>
+        public static void PartyCreateUpdate(Creature creature)
+        {
+            var packet = new Packet(Op.PartyCreateUpdate, 0);
+
+            packet.PutLong(creature.EntityId);
+            packet.PutInt(creature.PartyPosition);
+            packet.PutLong(creature.EntityId);
+            packet.PutString(creature.Name);
+            packet.PutByte(1);
```

Comment:
A byte that looks like a bool before a location usually means whether the location should be sent or not.